client/src/components/AuthClient.ts

```diff
@@ -237,6 +241,8 @@ const postAuthRequest = async (
   try {
     if (result.ok) {
       return await result.json();
+    } else {
```

Comment:
don't know how this else clause got removed, but this is necessary for any non-200 result status code and text to be returned. 

checked that returning the non-200 result json works on other `AuthClient` functions that use `postAuthRequest.`

Reply:
oh yeah that's great to have for error handling